### DISCUSSION TITLE
Can I disable color for just one type of level and still provide a prefix?

### DISCUSSION THREAD

**Discussion by sheldonhull** [pterm/pterm#450] [Q&A]
I'm detecting if CI=true.
Then I want to use a custom logging prefix per the provider, in this case Azure Pipelines.

For the warning and error type I'm using a specific output type required and want to exclude any color as it's not read correctly. In all other cases I'm ok leaving the formatting up as the logs are a better experience with it. 

```go
	pterm.Error = *pterm.Error.WithShowLineNumber().WithLineNumberOffset(1) //nolint:reassign // changing prefix later, not an issue.
	pterm.Error.Prefix = pterm.Prefix{Text: "#vso[task.logissue type=error]", Style: &pterm.Style{}} // 👈 can I set "Raw" output for just the error prefix type, without disabling globally?

	pterm.Debug.Prefix = pterm.Prefix{Text: "##[debug]", Style: &pterm.Style{}}
	pterm.Warning.Prefix = pterm.Prefix{Text: "##vso[task.logissue type=warning]", Style: &pterm.Style{}}

```

**Comment by MarvinJWendt** [comment]
Hi @sheldonhull, I am not sure if I understand it correctly. When you're not providing a style, or an empty one, to the `Prefix.Style` and `MessageStyle`, no color should be printed.

So are you looking for this?: 

```go
pterm.Warning.MessageStyle = pterm.NewStyle()
```

---

```go
func main() {
	pterm.Warning.Prefix = pterm.Prefix{Text: "##vso[task.logissue type=warning]", Style: pterm.NewStyle()}
	pterm.Warning.MessageStyle = pterm.NewStyle()
	pterm.Warning.Println("Hello, World!")
}
```

![image](https://user-images.githubusercontent.com/31022056/221436254-44670ddd-9ed4-4f9e-a11d-47f4d5279f15.png)
